docs/devops-guide/manual.md

```diff
@@ -164,18 +164,34 @@ ln -s ../sites-available/jitsi.example.com jitsi.example.com
 ```
 
 ## Install Jitsi Videobridge
-Visit https://download.jitsi.org/jitsi-videobridge/linux to determine the current build number, download and unzip it:
+
+You can download binary packages for Debian/Ubuntu:
+* [stable](https://download.jitsi.org/stable/) ([instructions](https://jitsi.org/downloads/ubuntu-debian-installations-instructions/))
+* [testing](https://download.jitsi.org/testing/) ([instructions](https://jitsi.org/downloads/ubuntu-debian-installations-instructions-for-testing/))
+* [nightly](https://download.jitsi.org/unstable/) ([instructions](https://jitsi.org/downloads/ubuntu-debian-installations-instructions-nightly/))
+
+Or you can clone the Git repo and run the JVB from source using maven.
+
+Install JDK and Maven if missing:
+
 ```sh
-wget https://download.jitsi.org/jitsi-videobridge/linux/jitsi-videobridge-linux-{arch-buildnum}.zip
-unzip jitsi-videobridge-linux-{arch-buildnum}.zip
+apt-get install openjdk-8-jdk maven
 ```
 
-Install JRE if missing:
-```
-apt-get install openjdk-8-jre
+Clone source from Github repo:
+
+```sh
+cd /srv
```

Comment:
I think this `cd /srv` is not necessary , but it should be mentioned that the user should create an empty folder where the cloned repository can be downloaded to.

Reply:
I agree with you